pkg/cluster/pod.go

```diff
@@ -67,7 +67,7 @@ func (c *Cluster) markRollingUpdateFlagForPod(pod *v1.Pod, msg string) error {
 		return fmt.Errorf("could not form patch for pod's rolling update flag: %v", err)
 	}
 
-	err = retryutil.Retry(c.OpConfig.PatroniAPICheckInterval, c.OpConfig.PatroniAPICheckTimeout,
+	err = retryutil.Retry(c.OpConfig.ResourceCheckInterval, c.OpConfig.ResourceCheckTimeout,
```

Comment:
what about the timeouts ?  PatroniAPICheckTimeout = 5s , ResourceCheckTimeout = 10 minutes

Reply:
we can go back to hard-coded timeouts again. Trying to patch for 10 minutes might too long, I agree.